Apps/phskypebusiness/readme.html

```diff
@@ -14,7 +14,7 @@ <h2>Registering the app on Azure portal</h2>
 <ol>
 	<li>Go to Azure Active Directory → App Registrations and click on <b>+New Application Registration</b>.</li>
 	<li>Give a name, select type as <b>“Web App/API”</b> and provide a sign-on URL.</li>
-	<li>After saving the app, open Manifest and change the field <b>‘oauth2AllowImplicitFlow’</b> from <i>false</i> to <i>true</i> and Save Manifest.</li>
+	<li>After saving the app, open Manifest, and change the field <b>‘oauth2AllowImplicitFlow’</b> from <i>false</i> to <i>true</i> and Save Manifest.</li>
```

Comment:
Since you added the comma, also remove the "and" right after it since you have one before "Save Manifest".

Reply:
Done